src/main/java/uk/ac/ebi/embl/api/validation/dao/EraproDAOUtilsImpl.java

```diff
@@ -108,9 +132,20 @@ public static boolean isNullValue(String qualifierValue) {
 	}
 	
 
-	public EraproDAOUtilsImpl (Connection connection)
+	public EraproDAOUtilsImpl (Connection connection, String biosamplesWebinUsername, String biosamplesWebinPassword)
```

Comment:
We need four inputs:
- bioSampleUserName( Webin-40894 )
- bioSamplePassword( Webin superuser password stored in BioSamples )
- webinUserName( su-Webin-N where Webin-N is the submittters submission account Id )
- webinPassword( Webin superuser password stored in ERAPRO )